src/bridge/cockpitpeer.c

```diff
@@ -400,7 +401,19 @@ on_other_control (CockpitTransport *transport,
           cockpit_transport_send (transport, NULL, reply);
           g_bytes_unref (reply);
         }
+    }
+
+  /* cockpit-bridge --privileged expects to receive a copy of our stderr */
+  else if (g_str_equal (command, "send-stderr") &&
+           cockpit_json_get_bool (self->config, "privileged", FALSE, &privileged) && privileged)
+    {
+      CockpitPipe *pipe = cockpit_pipe_transport_get_pipe (COCKPIT_PIPE_TRANSPORT (transport));
+
+      int out_fd;
+      g_object_get (pipe, "out-fd", &out_fd, NULL);
 
+      bool success = cockpit_socket_send_fd (out_fd, STDERR_FILENO);
+      g_assert (success);
```

Comment:
Maybe this should be less brutal.  I can't imagine why it would ever fail, but if it does, probably g_critical() would be nicer.  In that case, we should also definitely close the connection as well.